src/structures/Message.js

```diff
@@ -557,10 +557,10 @@ class Message extends Base {
    * @readonly
    */
   get deletable() {
-    return Boolean(
+    return (
```

Comment:
Once again this cannot be removed otherwise it might return undefined due to the last check being possibly undefined

Reply:
Can be replaced with `!!` to keep consistency, same for the review below.

Reply:
Yes that's what I meant